src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Machine> {
+    @Override
+    public List<Machine> get() {
+        List<Machine> excavators = new ArrayList<>();
```

Comment:
ExcavatorProducer should return the list of only Excavators. If we have Machine list we could put there other machine children and ExcavatorProducer will return list which contains not only excavators. And this will be wrong work of ExcavatorProducer .

Reply:
Done